src/main/scala/com/fulcrumgenomics/commons/io/Io.scala

```diff
@@ -46,15 +47,28 @@ trait IoUtil {
 
   /** How large a buffer should be used when buffering operations. */
   def bufferSize: Int = 32 * 1024
+  def compressionLevel: Int = 5
```

Comment:
Could you just add a docstring for this please?

Reply:
A docstring has been added!